.github/workflows-src/steps.lib.yml

```diff
@@ -100,7 +100,11 @@ run: |
 #@ def steps_inttest():
 name: "Integration Tests"
 shell: bash
-run: SHELL='' go test -timeout 20m ./test/integration -v
+run: |
```

Comment:
Is it possible to make this smarter like this:

We look for all labels that are specified and start with `int-test-only: <label>` and then add the `<label>` to `TEST_SUITE_TAGS`? Potentially several ones?